src/libs/ReportUtils.ts

```diff
@@ -1667,6 +1667,10 @@ function canAddOrDeleteTransactions(moneyRequestReport: OnyxEntry<Report>): bool
         return false;
     }
 
+    if (currentUserAccountID !== moneyRequestReport?.managerID && currentUserAccountID !== moneyRequestReport?.ownerAccountID) {
```

Comment:
I don't think we should have this check here. We should do the check in `canRequestMoney`. This function's only responsibility is to verify whether or not the report itself supports having transactions added/deleted. It doesn't check permissions.

Reply:
I would also only run this check if the report is an IOU.